### PR TITLE
CE-1634 Redesign the subpages

### PR DESCRIPTION
**Pull request by adamkarminski**
@lukaszkonieczny

### CODE REVIEW

File: extensions/wikia/Insights/models/InsightsWantedpagesModel.php [hunk 1]
```diff
@@ -35,4 +42,10 @@ public function isItemFixed( Article $article ) {
 		}
 		return false;
 	}
+
+	private function makeWlhLink( $title, $result ) {
```
Comment (lukaszkonieczny) [inline extensions/wikia/Insights/models/InsightsWantedpagesModel.php]: Please add function description

**Comment by lgarczewski** [comment]
LGTM.

File: extensions/wikia/Insights/styles/insights.scss [hunk 2]
```diff
@@ -1,22 +1,60 @@
...
+	.insights-nav-item.active {
+		background-color: $insights-background-grey;
+	}
 }
 
-.insights-icon-orphan {
-	background-image: url('/extensions/wikia/Insights/images/trending.png'); /* inline */
+.insights-dark {
+	.insights-icon-uncategorizedpages {
+		background-image: url('/extensions/wikia/Insights/images/uncategorized_dark.svg'); /* inline */
```
Comment (lukaszkonieczny) [inline extensions/wikia/Insights/styles/insights.scss]: Could we take benefits from svg and just set a color?

File: extensions/wikia/Insights/templates/Insights_index.php [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 		<h1><?= wfMessage( 'insights-landing-title' )->escaped() ?></h1>
-		<p><?= wfMessage( 'insights-landing-lead' )->escaped() ?></p>
+		<p><?= wfMessage( 'insights-landing-lead' )->parse() ?></p>
 	</div>
 	<div class="insights-landing-nav">
 		<ul class="insights-landing-nav-list">
 			<?php foreach( InsightsHelper::$insightsMessageKeys as $key => $messages ) : ?>
-				<a href="<?= InsightsHelper::getSubpageLocalUrl( $key ) ?>">
-					<li class="insights-landing-nav-item insights-icon-<?= $key ?>">
+				<li class="insights-landing-nav-item insights-icon-<?= strtolower( $key ) ?>">
```
Comment (lukaszkonieczny) [inline extensions/wikia/Insights/templates/Insights_index.php]: TODO: remove strtolower - it's already done in different PR

File: extensions/wikia/Insights/templates/Insights_subpageList.php [hunk 4]
```diff
@@ -1,7 +1,8 @@
-<div class="insights-container-nav">
+<div class="insights-container-nav <?= $themeClass ?>">
 	<ul class="insights-nav-list">
 		<? foreach( $data['messageKeys'] as $key => $messages ) : ?>
-			<li class="insights-nav-item">
+			<?php $subpage == $key ? $class = 'active' : $class = '' ?>
+			<li class="insights-nav-item insights-icon-<?= strtolower( $key ) ?> <?= $class ?>">
```
Comment (lukaszkonieczny) [inline extensions/wikia/Insights/templates/Insights_subpageList.php]: TODO: remove strtolower - it's already done in different PR

File: extensions/wikia/Insights/styles/insights.scss (hunk 2 above)
Comment (adamkarminski) [inline extensions/wikia/Insights/styles/insights.scss]: Discussed, it's a NTH and not crucial in the beta version.

File: extensions/wikia/Insights/models/InsightsWantedpagesModel.php (hunk 1 above)
Comment (adamkarminski) [inline extensions/wikia/Insights/models/InsightsWantedpagesModel.php]: TBDone before the final PR to dev
